src/hooks/core/chat/useFetchPostNotificationHook.ts

```diff
@@ -11,7 +11,7 @@ const useFetchPostNotificationHook = () => {
     if (!localDb) return;
     const allPromises: Promise<void>[] = notifications.map((notification) => {
       const channelList = fromNotificationAPI(notification);
-      return channelList.saveIfLatest(localDb);
+      return channelList.save(localDb);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The method `saveIfLatest` has been replaced with `save`. This could potentially lead to data inconsistency if older notifications overwrite newer ones. Please ensure that the `save` method includes a check for the latest data before saving, or consider reverting back to using `saveIfLatest`.
```diff
- return channelList.save(localDb);
+ return channelList.saveIfLatest(localDb);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->